app/src/hooks/cb_api/useXrefApi.tsx

```diff
@@ -29,8 +31,32 @@ export const useXrefApi = (axios: AxiosInstance) => {
     return data;
   };
 
+  /**
+   * Get collection (ie. ecological) units that are available for a given taxon
+   *
+   * @param {number} tsn
+   * @return {*}  {Promise<ICollectionCategory[]>}
+   */
+  const getCollectionCategories = async (tsn: number): Promise<ICollectionCategory[]> => {
```

Comment:
maybe rename to getTaxonCollectionCategories?

Reply:
I think renaming this makes sense if we would ever have a frontend hook that would retrieve all collection categories

Reply:
Renamed to `getCollectionCategoriesForTSN`.

Reply:
Happy to change the name to `getTsnCollectionCategories` if you want though.

Made it feel like the collection categories belonged to the TSN, which is maybe true, but they are their own thing, just filtered by TSN. Dunno.